enterprise/admin/views.py

```diff
@@ -250,11 +251,13 @@ def get_enterprise_customer_user_queryset(self, search_keyword, customer_uuid):
             customer_uuid (str): A unique identifier to filter down to only users linked to a
             particular EnterpriseCustomer.
         """
-        # TODO: Add paging so the admin view does not time out.
-        # For now, we will allow search, but will return an empty list
-        # if no search keyword is provided.
-        learners = []
-        if search_keyword is not None:
+        paginate_by = 10
```

Comment:
lets pass this variable as optional argument in the method (also include it in arguments docstring) instead of hardcoding it here
```python
 def get_enterprise_customer_user_queryset(self, request, search_keyword, customer_uuid, page_size=10):

paginate_by = page_size
```